packages/components/src/vue/vuex-modules/data/lexis.js

```diff
@@ -523,6 +541,87 @@ export default class Lexis extends Module {
     store.commit('lexis/hideCedictNotification')
     return result
   }
+
+  // TODO: Should be moved behind the GraphQL facade
+  static async getLemmaTranslations ({ homonym, browserLang, clientId, source }) {
+    const adapterTranslationRes = await ClientAdapters.lemmatranslation.alpheios({
+      method: 'fetchTranslations',
+      clientId: clientId,
+      params: {
+        homonym,
+        browserLang
+      }
+    })
+    /*
+    If the request obtained any translations they will be attached
+    to the `translation` prop of the corresponding lexeme's lemma object.
+    The `adapterTranslationRes` object will contain only errors, if any.
+     */
+    if (adapterTranslationRes.errors.length > 0) {
+      adapterTranslationRes.errors.forEach(error => Logger.getInstance().log(error.message))
+    }
+    // Suppress events that will trigger UI messages if source is wordlist
+    if (source !== LexicalQuery.sources.WORDLIST) {
+      LexicalQuery.evt.LEMMA_TRANSL_READY.pub(homonym)
+    } else {
+      LexicalQuery.evt.WORDLIST_UPDATE_LEMMA_TRANSL_READY.pub(homonym)
+    }
+    return homonym
+  }
+
+  // TODO: Should be moved behind the GraphQL facade
+  static async getWordUsageExamples ({ homonym, paginationAuthMax, clientId }) {
```

Comment:
I think we should have an issue for further refactoring here. This method is only called if the `wordUsageExamplesON` setting was set to 'onLexicalQuery'. If it's set instead to 'onDemand', the still-existing LexicalQuery.getWordUsage data method is called instead. As we have discussed previously, I would like for all resource queries to be able to set as 'pre-cache' or 'on-demand' and for that to trigger when they are called, but we should not have two separate methods for calling them depending upon when they are called.  This is fine for this PR, but we do need to refactor it.

Reply:
Agree, I've created an issue to track this: #570.